source/sass/mixins/_typography.scss

```diff
@@ -120,5 +120,24 @@
 @mixin list-style-none() {
   @include padding(0, inline-start);
   // Overflow auto is so the bullet will be still read from a screen reader and will push the bullet off screen
-  overflow: auto;
+  overflow-x: auto;
+}
+
+@mixin comma-separator() {
+  &:after {
+    display: inline;
```

Comment:
Can't remember why this was here, for older browsers?

Reply:
Not sure where this came from, doesn't seem to do anything. Removing.